extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -79,13 +79,13 @@ export class DraggableDrawer extends AMP.BaseElement {
     /** @private {!Array<!Element>} AMP components within the drawer. */
     this.ampComponents_ = [];
 
-    /** @private {?Element} */
+    /** @protected {?Element} */
```

Comment:
Are these actually used by a subclass?

Reply:
They're used in the same directory (`extensions/amp-story/1.0/`).

Reply:
That's not a protected property then. Protected should only apply to subclass relationships, not file-level/dir-level access.

Reply:
My mistake actually, this should be `@protected`. One class is extending this one and using `this.containerEl_`.